index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Intuit Accounts - Sing Up</title>
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <nav>
+    <p class="alText">Already have an Intuit account?</p>
+    <a class="fPass" href="#">I forgot my User ID or Password </a>
+    <button class="topSign" type="submit"><i class="fa fa-lg fa-lock"></i>Sign In</button>
+  </nav>
```

Comment:
This part is still not looking as the original one. The [previous reviewe](https://github.com/SigmaSam/DisplayInputData-Microverse-Project-2/pull/1)r mentioned this but you didn't change it.